pkg/subctl/cmd/resource.go

```diff
@@ -128,16 +128,16 @@ func getBrokerRestConfigFromSubmariner(submarinerClient *subOperatorClientset.Cl
 			Group:    submarinerv1.SchemeGroupVersion.Group,
 			Version:  submarinerv1.SchemeGroupVersion.Version,
 			Resource: "clusters",
-		})
+		}, submariner.Spec.BrokerK8sRemoteNamespace)
 
-	return restConfig, err
+	return restConfig, submariner.Spec.BrokerK8sRemoteNamespace, err
 }
 
-func getBrokerRestConfigFromServiceDisc(submarinerClient *subOperatorClientset.Clientset) (*rest.Config, error) {
+func getBrokerRestConfigAndNamespaceFromServiceDisc(submarinerClient *subOperatorClientset.Clientset) (*rest.Config, string, error) {
 	serviceDisc, err := submarinerClient.SubmarinerV1alpha1().ServiceDiscoveries(OperatorNamespace).
 		Get(names.ServiceDiscoveryCrName, v1opts.GetOptions{})
 	if err != nil {
-		return nil, errors.WithMessage(err, "error obtaining the ServiceDiscovery resource")
+		return nil, "", errors.WithMessage(err, "error obtaining the ServiceDiscovery resource")
```

Comment:
```suggestion
		return nil, "", errors.WithMessage(err, "Error obtaining the ServiceDiscovery resource")
```

Reply:
Actually `error` strings shouldn't be [capitalized](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).

Reply:
But we have them in subctl, so it should be uniformed

Reply:
Which places? I'm glancing thru some of the code and don't see capitalized error strings. There may be some but they should be changed.